frontend/public/components/factory/table.tsx

```diff
@@ -455,6 +476,7 @@ export const Table = connect<
       sortList: PropTypes.func,
       onSelect: PropTypes.func,
       scrollElement: PropTypes.oneOf([PropTypes.object, PropTypes.func]),
+      kind: PropTypes.string, // for column management should use gvk
```

Comment:
Call this something other than `kind` if it's gvk.

It might be better to give this a specific property like `columnManagementID` or something to make it really clear what's it for. Then you don't need the comment.

Reply:
replaced kind with columnManagementID throughout. Thanks!